.github/workflows/publish.yaml

```diff
@@ -79,7 +78,7 @@ jobs:
           echo "Tagging with ${{ inputs.image-tag }}"
           echo ${{ secrets.GITHUB_TOKEN }} | docker login $IMAGE_REGISTRY -u $GITHUB_ACTOR --password-stdin
           docker pull $API_IMAGE
-          echo $(git log -n 1 --format=format:'%h% %>(12) %cs %>(12,mtrunc)%aN%  %s %D') > ./api/src/version.txt
+          echo "$(git log -n 1 --format=format:'%h% %>(12) %cs %aN%  %s %d')"
```

Comment:
Not much value echoing this in the CI only. Should be put into a file that the API writes out to logs at startup

Reply:
Then I misunderstood something. Fixed.